avaframe/out3Plot/statsPlots.py

```diff
@@ -205,7 +185,7 @@ def plotHistCDFDiff(dataDiffPlot, ax1, ax2, insert='True', title=['', '']):
 
     ax2.set_xlim([-sortedDiffPlot[ind], sortedDiffPlot[ind]])
     width = diffMax - diffMin
-    stepWidth = 2*sortedDiffPlot[ind]/50 # 50 bins in the [-2sigma,+2sigma] interval
+    stepWidth = 2*sortedDiffPlot[ind]/50     # 50 bins in the [-2sigma,+2sigma] interval
```

Comment:
What is the reasoning for 50 bins? Should this go to the ini?

Reply:
this parameter and the values for the centiles are now in plotUtilsCfg.ini